### PR TITLE
Create codeql-analysis.yml

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

**Comment by florian-vuillemot** [comment]
@dinhlongviolin1 @FabienLelaquais @FredLL-Avaiga The code Scanning raises some warning/error. Could you please have a look?

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
```
Comment (FredLL-Avaiga) [inline .github/workflows/codeql-analysis.yml:23]: typescript ?

Comment (FabienLelaquais) [reply .github/workflows/codeql-analysis.yml:23]: It looks like this is transparent for CodeQL...

**Review by FabienLelaquais — APPROVED** [review]
👍

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (FredLL-Avaiga) [reply .github/workflows/codeql-analysis.yml:23]: just saying ;-)
I've read in the doc that typescript is supported

Comment (florian-vuillemot) [reply .github/workflows/codeql-analysis.yml:23]: He does both at the same time.

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**